java/cuvs-java/src/main/java22/com/nvidia/cuvs/internal/common/Util.java

```diff
@@ -120,6 +122,18 @@ public static void cudaMemcpy(MemorySegment dest, MemorySegment src, long numByt
     Util.cudaMemcpy(dest, src, numBytes, CudaMemcpyKind.INFER_DIRECTION);
   }
 
+  /**
+   * Helper to get the default CUDA stream associated with a {@link CuVSResources}
+   */
+  public static MemorySegment getDefaultStream(CuVSResources resources) {
```

Comment:
I am hesitant to call this the "default" stream.  In CUDA parlance, the default-stream is one of the following:
1. the single stream on which all kernels are scheduled by default, OR
2. when per-thread default streams are enabled, the stream on which kernels are scheduled, for a specific CPU thread.

This variable name overloads the parlance.  Could we consider calling this function `getStream(CuVSResources)`?

Alternatively, could we please add this as a method called `getStream()` in `CuVSResources`?